docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -77,19 +82,24 @@ In addition to our documentation site EdgeX foundry also releases our API specif
 
 The EdgeX Foundry community has a set of tests we maintain to do regression testing during development this framework is tracking the `master` branch of the components of EdgeX. At the time of release we will update the testing frameworks to point at the released Github tags and add a version tag to the testing frameworks themselves. This creates a snapshot of testing framework at the time of release for validation of the official release.
 
+### GitHub Release Artifacts
+
+*Tied to Code Release?* Yes
+
+GitHub release functionality is utilized on some repositories to release binary artifacts/assets (e.g. zip/tar files). These are versioned with the semantic version and found on the repository's GitHub Release page under 'Assets'.
+
 ### Known Build Dependencies for EdgeX Foundry
 
 There are some internal build dependencies within the EdgeX Foundry organization. When building artifacts for validation or a release you will need to take into the account the build dependencies to make sure you build them in the correct order.
 
 ![Known EdgeX Foundry Build Dependencies](0010/known-build-dependencies.png)
 
- - Edgex-go Snap: Has dependencies on app-service-configurable, device-virtual-go and support-rulesengine because they are included in the snap.
- - Application services have a dependency on the Application services SDK.
- - Go Device services have a dependency on the Go Device SDK.
- - C Device services have a dependency on the C Device SDK.
+- Application services have a dependency on the Application services SDK.
```

Comment:
technically it's called Application Functions SDK

Reply:
Also, beyond the SDKs there are numerous internal Go modules (e.g. go-mod-bootstrap, go-mod-core-contracts, ...). Should these also be mentioned here?

Reply:
I updated the Application Functions SDK phrasing.

I don't think we want to reference any build dependencies that are actually tracked within go.mod files or part of the source.

I believe Lisa's intent for this was to track things at a higher level that might not be included as part of the automation and be lost to tribal knowledge. (e.g. needing to build/release appfunctions-sdk-go before building/releasing application services.)